<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMenu</class>
 <widget class="QMainWindow" name="frmMenu">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario de Inicio</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivos">
    <property name="title">
     <string>Archivos</string>
    </property>
    <addaction name="actionCarga_de_Datos"/>
    <addaction name="actionConsulta_de_Datos"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCarga_de_Datos">
   <property name="text">
    <string>Carga de Datos</string>
   </property>
  </action>
  <action name="actionConsulta_de_Datos">
   <property name="text">
    <string>Consulta de Datos</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
